src/Raven.Studio/wwwroot/App/views/database/timeSeries/editTimeSeriesEntry.html

```diff
@@ -73,19 +73,37 @@ <h4 class="modal-title">Time Series Entry for <strong data-bind="text: documentI
                                     <div class="help-block margin-bottom" data-bind="validationMessage: values"></div>
                                 </div>
                             </div>
-                            <button class="btn btn-default btn-sm" data-bind="click: addValue">
+                            <button class="btn btn-default btn-sm" data-bind="click: addValue, disable: maxNumberOfValuesReached">
                                 <i class="icon-plus"></i>
                                 <span>Add Value</span>
                             </button>
                         </div>
                     </div>
                     <div class="form-group" data-bind="visible: $root.isAggregation">
                         <label class="control-label">&nbsp;</label>
-                        <div class="flex-horizontal bg-warning small padding padding-sm">
-                            <i class="icon-warning padding"></i>
-                            <div class="text-warning">
-                                Please note: above values are the results of a pre-defined aggregation policy.<br />
-                                Editing or deleting any value can impact other aggregations and queries.
+                        <div class="bg-warning text-warning padding padding-sm flex-horizontal">
+                            <div class="flex-start"><small><i class="icon-warning"></i></small></div>
+                            <div>
+                                <small>Please note: above values are the results of a pre-defined aggregation policy.</small><br>
+                                <small>Editing or deleting any value can impact other aggregations and queries.</small>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="form-group" data-bind="visible: maxNumberOfValuesReached">
+                        <label class="control-label">&nbsp;</label>
+                        <div class="bg-info text-info padding padding-xs flex-horizontal">
+                            <div class="flex-start"><small><i class="icon-info"></i></small></div>
+                            <div>
+                                <small>The maximum number of possible values (32) has been reached.</small>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="form-group" data-bind="visible: values().length > 5">
+                        <label class="control-label">&nbsp;</label>
+                        <div class="bg-warning text-warning padding padding-xs flex-horizontal">
+                            <div class="flex-start"><small><i class="icon-warning"></i></small></div>
+                            <div>
+                                <small>Note: Aggregated rollup data is Not created for a Time Series if an Entry has more than 5 values.</small>
```

Comment:
Please avoid using `Not`. If you need to emphasize this word please use strong or em.

Reply:
ok